src/betterproto/__init__.py

```diff
@@ -487,6 +490,51 @@ def decode_varint(buffer: bytes, pos: int) -> Tuple[int, int]:
             raise ValueError("Too many bytes when decoding varint.")
 
 
+def read_varint(stream: BinaryIO) -> Optional[int]:
```

Comment:
These are often called `load` (see `json.load`).

I feel that we should raise `EOFError` so that the user does not have to check for `None` in their code. They can loop until they hit `EOFError`. The return type can be `int`.

```suggestion
def load_varint(stream: BinaryIO) -> int:
```

Reply:
Changed